app/controllers/events_controller.rb

```diff
@@ -0,0 +1,68 @@
+class EventsController < ApplicationController
+  def index
+    @event = Event.all
+    @upcoming_event = Event.upcoming
+    @past_event = Event.past
+
+    @attendances = Attendance.all
+    @attendance = Attendance.new
+  end
+
+  def new
+    @event = Event.new
+  end
+
+  def create
```

Comment:
I'm unable to create an event:

![image](https://user-images.githubusercontent.com/22228397/64927695-c71abb80-d7d3-11e9-855a-01aec1856970.png)

Result:

![image](https://user-images.githubusercontent.com/22228397/64927702-d863c800-d7d3-11e9-9ebd-7f4af1af626f.png)

Output from terminal:

```
Started POST "/events" for ::1 at 2019-09-15 16:12:43 -0500
Processing by EventsController#create as HTML
  Parameters: {"utf8"=>"✓", "authenticity_token"=>"KgCn9khnAW9cCI9433FSan2Xvh531d93D91npPXYwrfPz/eBxWslLjenT+PZOrKVF7EY8fR7kUo/7avEV9DUWg==", "event"=>{"event_title"=>"Hello World", "description"=>"Lorem Ipsum", "date"=>"2019-10-10"}, "commit"=>"create"}
  User Load (0.2ms)  SELECT  "users".* FROM "users" WHERE "users"."id" = ? LIMIT ?  [["id", 1], ["LIMIT", 1]]
  ↳ app/helpers/sessions_helper.rb:9
   (0.1ms)  begin transaction
  ↳ app/controllers/events_controller.rb:18
   (0.1ms)  rollback transaction
  ↳ app/controllers/events_controller.rb:18
Redirected to http://localhost:3000/
Completed 302 Found in 8ms (ActiveRecord: 0.5ms)
```

Looking into your validations, I see this is because there's a 20 character minimum requirement for a description. If that's the case, it'd be nice if you can add a warning about this or an indication of it in the description.

[OPTIONAL]